climada/hazard/centroids/centr.py

```diff
@@ -950,7 +963,15 @@ def from_hdf5(cls, file_name):
                 # in previous versions of CLIMADA and/or geopandas,
                 # the CRS was stored in '_crs'/'crs'
                 crs = metadata.get("crs")
-                gdf = gpd.GeoDataFrame(store["centroids"], crs=crs)
+                gdf = gpd.GeoDataFrame(store["centroids"])
+                for xycol in metadata.get("xy_columns", []):
```

Comment:
Please also add a test with multiple xy_columns/wkb_columns to be stored and read.